lib/android-helpers.js

```diff
@@ -635,7 +635,9 @@ helpers.initDevice = async function (adb, opts) {
     await helpers.setMockLocationApp(adb, SETTINGS_HELPER_PKG_ID);
   }
 
-  await helpers.ensureDeviceLocale(adb, opts.language, opts.locale, opts.localeScript);
+  if (opts.language || opts.locale) {
```

Comment:
should it be `||` or `&&` ?

Reply:
`||` is fine here since if users set only `language` (or `locale`), then we should show a message in `ensureDeviceLocale` to guide them.
We can skip calling the `ensureDeviceLocale` when they do not specify both language and locale in their capability.